.devcontainer/devcontainer.json

```diff
@@ -1,137 +1,127 @@
-            "zsh": {
-              "path": "/usr/bin/zsh"
-            }
-          },
-  
-          // Uncomment if you want to use CaskaydiaCove Nerd Font as the default terminal font
-          "terminal.integrated.fontFamily": "CaskaydiaCove Nerd Font",
-  
-          // Uncomment if you want to disable the minimap view
-          "editor.minimap.enabled": false,
-  
-          // Recommended settings for the explorer pane
-          "explorer.sortOrder": "type",
-          "explorer.fileNesting.enabled": true,
-          "explorer.fileNesting.patterns": {
-            "*.bicep": "${capture}.json",
-            "*.razor": "${capture}.razor.css",
-            "*.js": "${capture}.js.map"
+    // Uncomment the below to install node.js
+    "ghcr.io/devcontainers/features/node:1": {
+      "version": "lts",
+      "nodeGypDependencies": true,
+      "nvmInstallPath": "/usr/local/share/nvm"
+    },
+    // Install common utilities
+    "ghcr.io/devcontainers/features/common-utils:1": {
+      "installZsh": true,
+      "installOhMyZsh": true,
+      "upgradePackages": true,
+      "username": "vscode",
+      "uid": "1000",
+      "gid": "1000"
+    }
+  },
+  "overrideFeatureInstallOrder": [
+    "ghcr.io/devcontainers/features/common-utils"
+  ],
+  // Configure tool-specific properties.
+  "customizations": {
+    // Configure properties specific to VS Code.
+    "vscode": {
+      // Add the IDs of extensions you want installed when the container is created.
+      "extensions": [
+        // Recommended extensions - GitHub
+        "cschleiden.vscode-github-actions",
+        "GitHub.vscode-pull-request-github",
+        // Recommended extensions - Azure
+        "Azurite.azurite",
+        "ms-azuretools.vscode-bicep",
+        // Recommended extensions - Collaboration
+        "eamodio.gitlens",
+        "EditorConfig.EditorConfig",
+        // "MS-vsliveshare.vsliveshare-pack",
+        "streetsidesoftware.code-spell-checker",
+        // Recommended extensions - .NET
+        "Fudge.auto-using",
+        "jongrant.csharpsortusings",
+        "kreativ-software.csharpextensions",
+        // Recommended extensions - Power Platform
+        // "microsoft-IsvExpTools.powerplatform-vscode",
+        // Recommended extensions - Markdown
+        "bierner.github-markdown-preview",
+        "DavidAnson.vscode-markdownlint",
+        "docsmsft.docs-linting",
+        "johnpapa.read-time",
+        "yzhang.markdown-all-in-one",
+        // Required extensions
+        "GitHub.copilot",
+        "ms-dotnettools.csharp",
+        "ms-vscode.PowerShell",
+        "ms-vscode.vscode-node-azure-pack",
+        "VisualStudioExptTeam.vscodeintellicode",
+        // .NET Interactive
+        "ms-dotnettools.csharp",
+        "ms-dotnettools.dotnet-interactive-vscode",
+        "ms-python.python",
+        "ms-toolsai.jupyter",
+        "ms-toolsai.jupyter-keymap",
+        "ms-toolsai.jupyter-renderers",
+        "ms-vscode.powershell",
```

Comment:
It's dup'd

Reply:
Fixed in this PR https://github.com/microsoft/dotnetconf-studentzone/pull/3 if you want to take a look @justinyoo